packages/e2e/test/web-extension/specs/wallet.spec.ts

```diff
@@ -48,12 +57,18 @@ describe('wallet', () => {
         await expect($(spanAddress)).toHaveTextContaining('addr');
       });
       it('can build and sign a transaction', async () => {
-        await $(btnSignAndBuildTx).click();
-        await browser.waitUntil(async () => {
-          const signature = await $(divSignature).getText();
-          return signature.length > 1;
-        });
+        await buildAndSign();
       });
+      it('can create new wallet and switch to it', async () => {
+        const prevWalletAddr = await $(spanAddress).getText();
+        await $(btnCreateWallet2).click();
+        await expect($(spanAddress)).not.toHaveTextContaining(prevWalletAddr);
+        await expect($(spanAddress)).toHaveTextContaining('addr');
+      });
+      it('can build and sign a transaction using the new wallet', async () => {
+        await buildAndSign();
+      });
+
```

Comment:
Let's extend the tests to better cover functionality of the wallet manager
```suggestion
it('can switch to another wallet', async () => {
it('can switch back to the first wallet', async () => {
```

Reply:
Added here: [deb1c66](https://github.com/input-output-hk/cardano-js-sdk/pull/502/commits/deb1c6617b48ac03ce5a59e0065db69a114cebee)